.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 
-php: [5.3, 5.4, 5.5, 5.6, hhvm]
+php: [5.5, 5.6, hhvm]
```

Comment:
well, your composer constraint **already** keeps the support for older Goutte versions. So I don't see why it would fail